src/snowcli/cli/loggers.py

```diff
@@ -1,32 +1,113 @@
+        possible_log_levels = [
+            logging.DEBUG,
+            logging.INFO,
+            logging.WARN,
+            logging.ERROR,
+            logging.CRITICAL,
+        ]
+        if self.file_log_level not in possible_log_levels:
+            raise InvalidLogsConfiguration(
+                f"Invalid 'level' value set in [logs] section: {config['level']}. "
+                f"'level' should be one of: {' / '.join(logging.getLevelName(lvl) for lvl in possible_log_levels)}"
+            )
+
+
+def remove_console_output_handler_from_logs() -> None:
+    logger = logging.getLogger("snowcli")
+    for handler in logger.handlers:
+        if handler.name == _CONSOLE_OUTPUT:
+            logger.removeHandler(handler)
+
+
+def add_console_output_handler_to_logs(
+    log_level: int, formatter: logging.Formatter
+) -> None:
+    logger = logging.getLogger("snowcli")
+    console = logging.StreamHandler()
+    console.set_name(_CONSOLE_OUTPUT)
+    console.setFormatter(formatter)
+    console.setLevel(log_level)
+    logger.addHandler(console)
 
 
 def create_loggers(verbose: bool, debug: bool):
-    """Creates a logger depending on the SnowCLI parameters
+    """Creates a logger depending on the SnowCLI parameters and config file.
     verbose == True - print info and higher logs in default format
     debug == True - print debug and higher logs in debug format
     none of above - print only error logs in default format
     """
-    logger = logging.getLogger("snowcli")
+    config = LogsConfig(debug=debug)
+
     if verbose and debug:
         raise typer.BadParameter("Only one parameter `verbose` or `debug` is possible")
     elif debug:
-        log_format = DEBUG_LOG_FORMAT
-        log_level = logging.DEBUG
+        console_log_format = DEBUG_LOG_FORMAT
+        console_log_level = logging.DEBUG
     elif verbose:
-        log_format = DEFAULT_LOG_FORMAT
-        log_level = logging.INFO
+        console_log_format = DEFAULT_LOG_FORMAT
+        console_log_level = logging.INFO
     else:
-        log_format = DEFAULT_LOG_FORMAT
-        log_level = logging.ERROR
+        console_log_format = DEFAULT_LOG_FORMAT
+        console_log_level = logging.ERROR
 
-    console = logging.StreamHandler()
-    console.setFormatter(logging.Formatter(log_format, "%Y-%m-%d %H:%M:%S"))
-    logger.addHandler(console)
-    logger.setLevel(log_level)
-    logging.getLogger("snowflake").setLevel(log_level)
+    if console_log_level < config.file_log_level:
+        global_log_level = console_log_level
+    else:
+        global_log_level = config.file_log_level
+
+    logger = logging.getLogger("snowcli")
+    logger.setLevel(global_log_level)
+    logging.getLogger("snowflake").setLevel(global_log_level)
+
+    add_console_output_handler_to_logs(
+        console_log_level, formatter=logging.Formatter(console_log_format, DATE_FORMAT)
+    )
+
+    if config.save_logs:
+        filename = config.path / "snowcli.log"
```

Comment:
Should it be a constant? Should we encapsulate this into a method?

Reply:
done :)
[named _DEFAULT_LOGS_FILENAME]